### ISSUE TITLE
NodeJS v12 support

### ISSUE THREAD

**Issue by vasev01** [zowe/zowe-cli-db2-plugin#29]
On a system with NodeJS v12, building the project from source results in the following error:
```
...
make: *** [Release/obj.target/odbc_bindings/src/odbc.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:203:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:272:12)
gyp ERR! System Linux 4.15.0-54-generic
gyp ERR! command "/usr/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "configure" "build" "--IS_DOWNLOADED=true" "--IBM_DB_HOME=/home/user/node12test/zowe-cli-db2-plugin/node_modules/ibm_db/installer/clidriver"
gyp ERR! cwd /home/user/node12test/zowe-cli-db2-plugin/node_modules/ibm_db
gyp ERR! node -v v12.6.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 

    at ChildProcess.exithandler (child_process.js:297:12)
    at ChildProcess.emit (events.js:203:13)
    at maybeClose (internal/child_process.js:1021:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5) {
  killed: false,
  code: 1,
  signal: null,
  cmd: 'node-gyp configure build  --IS_DOWNLOADED=true --IBM_DB_HOME="$IBM_DB_HOME"'
}
npm WARN @zowe/db2@3.0.1 requires a peer of @zowe/cli@^3.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.9 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.9: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ibm_db@2.5.0 install: `node installer/driverInstall.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the ibm_db@2.5.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/user/.npm/_logs/2019-07-12T13_50_52_176Z-debug.log
```

App | Version
-----|---------
Linux | 4.15.0 64bit
NodeJS | v12.6.0
NPM | 6.9.0

**Comment by vasev01** [comment]
The `ibm_db` library issuse https://github.com/ibmdb/node-ibm_db/issues/528

**Comment by t1m0thyj** [comment]
This was addressed in [v4.0.2](https://github.com/zowe/zowe-cli-db2-plugin/blob/master/CHANGELOG.md#402):
> Updated ibm_db dependency to add Node 12 support
